services/services.go

```diff
@@ -119,6 +120,11 @@ func (s *Service) PollAction() {
 	}
 }
 
+// PollStop does nothing in a Service
+func (s *Service) PollStop() {
```

Comment:
I really like that we added this to the interface... I think we can leverage it down the road to maybe simplify the quit logic we have.